package.json

```diff
@@ -12,9 +12,11 @@
     "type-check": "tsc",
     "lint": "eslint --ignore-path .gitignore \"src/**/*.+(ts|js|tsx)\"",
     "format": "prettier --ignore-path .gitignore \"src/**/*.+(ts|js|tsx)\" --write",
-    "test": "jest --passWithNoTests"
+    "test": "jest --passWithNoTests",
+    "prepare": "husky install"
   },
   "dependencies": {
+    "@googlemaps/polyline-codec": "1.0.25",
```

Comment:
There seem to be some changes, which doesn't belong here,

Reply:
correct